docs/tutorials/tutorial-query.md

```diff
@@ -162,150 +272,8 @@ The following results should be returned:
-└─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
-Retrieved 1 row in 0.03s.
-```
-
-
-## Native JSON queries
-
-Druid's native query format is expressed in JSON.
-
-### Native query via the console
-
-You can issue native Druid queries from the console's Query view.
-
-Here is a query that retrieves the 10 Wikipedia pages with the most page edits on 2015-09-12.
-
-```json
-{
-  "queryType" : "topN",
-  "dataSource" : "wikipedia",
-  "intervals" : ["2015-09-12/2015-09-13"],
-  "granularity" : "all",
-  "dimension" : "page",
-  "metric" : "count",
-  "threshold" : 10,
-  "aggregations" : [
-    {
-      "type" : "count",
-      "name" : "count"
-    }
-  ]
-}
-```
-
-Simply paste it into the console to switch the editor into JSON mode.
-
-![Native query](../assets/tutorial-query-07.png "Native query")
-
-
-### Native queries over HTTP
-
-We have included a sample native TopN query under `quickstart/tutorial/wikipedia-top-pages.json`:
-
-Let's submit this query to Druid:
-
-```bash
-curl -X 'POST' -H 'Content-Type:application/json' -d @quickstart/tutorial/wikipedia-top-pages.json http://localhost:8888/druid/v2?pretty
-```
-
-You should see the following query results:
-
-```json
-[ {
-  "timestamp" : "2015-09-12T00:46:58.771Z",
-  "result" : [ {
-    "count" : 33,
-    "page" : "Wikipedia:Vandalismusmeldung"
-  }, {
-    "count" : 28,
-    "page" : "User:Cyde/List of candidates for speedy deletion/Subpage"
-  }, {
-    "count" : 27,
-    "page" : "Jeremy Corbyn"
-  }, {
-    "count" : 21,
-    "page" : "Wikipedia:Administrators' noticeboard/Incidents"
-  }, {
-    "count" : 20,
-    "page" : "Flavia Pennetta"
-  }, {
-    "count" : 18,
-    "page" : "Total Drama Presents: The Ridonculous Race"
-  }, {
-    "count" : 18,
-    "page" : "User talk:Dudeperson176123"
-  }, {
-    "count" : 18,
-    "page" : "Wikipédia:Le Bistro/12 septembre 2015"
-  }, {
-    "count" : 17,
-    "page" : "Wikipedia:In the news/Candidates"
-  }, {
-    "count" : 17,
-    "page" : "Wikipedia:Requests for page protection"
-  } ]
-} ]
-```
-
-
 ## Further reading
 
-The [Queries documentation](../querying/querying.md) has more information on Druid's native JSON queries.
+See the [Queries documentation](../querying/querying.md) for more information on Druid native query.
 
-The [Druid SQL documentation](../querying/sql.md) has more information on using Druid SQL queries.
+See the [Druid SQL documentation](../querying/sql.md) for more information on using Druid SQL queries.
```

Comment:
Should the order of these links be flipped since it makes SQL more prominent than native queries?